### ISSUE TITLE
cmake fails when MPI not found

### ISSUE THREAD

**Issue by tblum** [bh107/bohrium#30]
According to the doc http://bohrium.readthedocs.org/installation/linux.html#install-from-source-package
MPI is optional. But cmake wil fail if MPI is not available:
```
-- Could NOT find MPI_C (missing:  MPI_C_LIBRARIES MPI_C_INCLUDE_PATH) 
-- Could NOT find MPI_CXX (missing:  MPI_CXX_LIBRARIES MPI_CXX_INCLUDE_PATH) 
CMake Error at vem/cluster/CMakeLists.txt:13 (message):
   MPI not found! Set VEM_CLUSTER=OFF or install MPI.
```
Would be nice if it really was optional. Otherwise it should be listed as a required package.

**Comment by kenkendk** [comment]
It used to be that the install would continue if MPI was not found.
What is the motivation for requiring the user to set the CMake flags?

**Comment by madsbk** [comment]
Fixed in 3a79fa6a65ac5fe242a246a4da67062875f0d9f2, which makes ``VEM_CLUSTER=OFF`` the default option.
